gltest_cli/config/user.py

```
@@ -46,11 +62,24 @@ def get_default_user_config() -> UserConfig:
+                if var_name is None:
+                    raise ValueError(
+                        f"Invalid environment variable pattern: {m.group(0)}"
+                    )
+                var_value = os.getenv(var_name)
+                if var_value is None:
+                    raise ValueError(
+                        f"Environment variable {var_name} is not set, please check your environment file"
+                    )
+                return var_value
+            except IndexError:
+                raise ValueError(f"Invalid environment variable pattern: {m.group(0)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)